src/modm/platform/core/cortex/module.md

```diff
@@ -60,6 +60,15 @@ It is strongly recommended to only read/write registers in this function, and
 perhaps even write this function in Assembly if deemed necessary.
 
 
+### Cache Initialization
+
+For Cortex-M7 devices, both the I-Cache and D-Cache are enabled by default with
+a write-through policy to significantly improve performance. However, it is
```

Comment:
The default SRAM cache policy on STM32 M7 devices is write-back write-allocate, not write-through.

From AN4839:
![Screenshot_20241111_190138](https://github.com/user-attachments/assets/b5a552fc-25b7-47bc-a0eb-a4b131272143)

Also write-through is broken on half of the H7 devices so we shouldn't change the code to enable it:
![Screenshot_20241111_190712](https://github.com/user-attachments/assets/e7ab97d2-5b1a-4fc9-b98e-88dc5906d542)